packages/api/tests/models/test-pdrs-model.js

```diff
@@ -26,12 +25,14 @@ const createPdrMessage = ({
   numFailedExecutions = 0,
   numRunningExecutions = 0,
   createdAtTime = Date.now(),
+  execution = randomId('execution'),
   providerId = 'prov1',
+  stateMachine = randomId('pdr'),
```

Comment:
It doesn't matter greatly, but I assume you meant to do something like `randomId('stateMachine')` here